src/main/java/database/helper/Settings.java

```diff
@@ -83,4 +82,29 @@ Path getFilePath(String directoryName) throws CreationDatabaseException {
             throw new CreationDatabaseException("Directory cannot be created: " + e.getMessage());
         }
     }
+
+    private Properties loadProperties() {
```

Comment:
Let's load properties only once in constructor. Reading properties file every time can affect performance of the application. Try to load properties and set default values in `Properties` object if file or specific cannot be read.